nipoppy_cli/nipoppy/data/examples/sample_invocations/RABIES-0.5.1-preprocess.json

```diff
@@ -0,0 +1,7 @@
+{
+    "plugin": "MultiProc",
+    "local_threads": 4,
+    "anatomical_resampling": "0.3x0.3x0.3",
```

Comment:
Is this supposed to be the defaults? This is to speed test the pipeline, but in real application we would lost loads of data. Do not provide any additional defaults for preprocessing.